cmd/jwt.go

```diff
@@ -55,27 +55,21 @@ func cachedAuthenticateNode(ttl time.Duration) func(accessKey, secretKey, audien
 	type key struct {
 		accessKey, secretKey, audience string
 	}
-	type value struct {
-		created time.Time
-		res     string
-		err     error
-	}
-	cache, err := lru.NewARC(100)
-	if err != nil {
-		logger.LogIf(GlobalContext, err)
-		return authenticateNode
-	}
-	return func(accessKey, secretKey, audience string) (string, error) {
+
+	cache := expirable.NewLRU[key, string](100, nil, ttl)
```

Comment:
Generally not a fan of plain LRU, since it can be wiped with a single sweep. But I see the TwoQueueCache doesn't have ttl built in, so I guess it is ok here.